frontend/src/components/assessments/EditStatusPopover.tsx

```diff
@@ -58,6 +59,10 @@ const EditStatusPopover = ({
                 closePopover={onClose}
               />
             )}
+            <DuplicateButton
```

Comment:
I think we only want the `Duplicate` feature to show for `DRAFT` and `PUBLISHED` assessments, would you be add that check in? :D